index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href='http://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="profile">
+        <img class="pic" src="./images/TimScott.jpg" alt="Tim Scott" />
+        <div class="name">Tim Scott</div>
+        <div class="title">Senate (R-SC 1st District) - 2014</div>
+        <div class="separator"></div>
```

Comment:
I would have wrapped the bottom 2 elements in a container and added a border, but this'll do